doc/tokens/tokentypes/questionnaire.rst

```diff
@@ -8,15 +8,21 @@ Questionnaire Token
 The administrator can define a list of questions and also how many answers to
 the questions a user needs to define.
 
-During enrollment of such a *question* type token the user answers at least as
-many questions as specified with answers only he knows.
+During enrollment of such a *questionnaire* type token, the user answers at least as
+many questions as specified by the administrator with answers only he knows.
+
+.. figure:: images/enroll_questionnaire.png
+   :width: 500
 
 This token is a challenge response token.
-During authentication the user must give the token PIN and the a random
+During authentication the user gives the token PIN and a random
 question from the answered question is chosen. The user has to answer with
 the same answer he defined earlier.
 
-.. note:: If the administrator changes the questions _after_ a token was
+.. note:: By default, no questions are defined, so the administrator has to setup those
+   in "Config->Tokens->Questionnaire" before a questionnaire token can be rolled out successfully.
```

Comment:
Link to config

Reply:
no config section there yet. Should we do one? I think it is not neccessary.